### PR TITLE
Get drops entitlements

### PR DESCRIPTION
**Pull request by boyan-improbable**
Introduce the functionality to query for entitlements given to users by the Twitch Drops product offering.

More specifically: https://dev.twitch.tv/docs/api/reference#get-drops-entitlements

### CODE REVIEW

File: drops.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
+package helix
+
+// GetDropEntitlementsParams ...
+type GetDropEntitlementsParams struct {
+	EntitlementID string `query:"id"`
```
Comment (nicklaw5) [inline drops.go:5]: Please change `EntitlementID` to `ID` to be consistent with other structs in this library.

File: drops.go [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+	UserID        string `query:"user_id"`
+	GameID        string `query:"game_id"`
+	After         string `query:"after"`
+	First         int    `query:"first,20"` // Limit 100
+}
+
+// Entitlement ...
+type Entitlement struct {
+	EntitlementID string `json:"id"`
+	BenefitID     string `json:"benefit_id"`
```
Comment (nicklaw5) [inline drops.go:15]: Please change `EntitlementID` to `ID` to be consistent with other structs in this library.

**Review by nicklaw5 — CHANGES_REQUESTED** [review]
Thanks for the PR @boyan-improbable.

Just the two comments I'd like actioned and then this can be merged.

File: drops.go (hunk 1 above)
Comment (boyan-improbable) [reply drops.go]: Thanks for pointing it out - fixed now.

File: drops.go (hunk 2 above)
Comment (boyan-improbable) [reply drops.go:15]: Fixed this one as well 👍

**Comment by nicklaw5** [comment]
Thanks @boyan-improbable :rocket: 

I've published [`v1.0.1`](https://github.com/nicklaw5/helix/releases/tag/v1.0.1) which includes this addition.

**Pull request merged**